File: src/main/java/miniservice/TableCountRecord.java
package miniservice;

import javafx.beans.property.IntegerProperty;
import javafx.beans.property.StringProperty;
import javafx.scene.control.Button;

public class TableCountRecord {
    private StringProperty tableNameProperty;
    private IntegerProperty countProperty;
    private Button deleteButton;

    public StringProperty tableNameProperty() {
	return tableNameProperty;
    }

    public void setTableNameProperty(StringProperty tableNameProperty) {
	this.tableNameProperty = tableNameProperty;
    }

    public IntegerProperty countProperty() {
	return countProperty;
    }

    public void setCountProperty(IntegerProperty countProperty) {
	this.countProperty = countProperty;
    }

    public Button getDeleteButton() {
	return deleteButton;
    }

    public void setDeleteButton(Button deleteButton) {
	this.deleteButton = deleteButton;
    }
}

File: src/main/java/tdc/TableCountRecord.kt
package tdc

import javafx.beans.property.IntegerProperty
import javafx.beans.property.StringProperty
import javafx.scene.control.Button
import org.springframework.stereotype.Component

@Component
class TableCountRecord {
    private var tableNameProperty: StringProperty? = null
    private var countProperty: IntegerProperty? = null
    var deleteButton: Button? = null
    fun tableNameProperty(): StringProperty? {
        return tableNameProperty
    }

    fun setTableNameProperty(tableNameProperty: StringProperty?) {
        this.tableNameProperty = tableNameProperty
    }

    fun countProperty(): IntegerProperty? {
        return countProperty
    }

    fun setCountProperty(countProperty: IntegerProperty?) {
        this.countProperty = countProperty
    }
}